app/views/publish_name_email/build.html.erb

```diff
@@ -11,7 +11,7 @@
   links.push([:email_links_show_object.t(type: :name),
     "#{MO.http_domain}/name/show_name/#{@name.id}"])
   links.push([:email_links_email_publisher.t,
-    "#{MO.http_domain}/observer/show_user/#{@publisher.id}"])
+    "#{MO.http_domain}/users/#{@publisher.id}"])
```

Comment:
I'm confused.  With name and image controllers the path is just "/name" and "/image", respectively.  Why is it plural for the user controller?  Is this something to do with user controller routing differently than the other controllers?

Reply:
It's like Joe's "/herbaria/" -- it's because they're the Rails "resource" CRUD routes. They seem to default to the plural.

Reply:
Ah!  Understood.  Thank you!